org/issue.ts

```diff
@@ -4,27 +4,91 @@ const gh = danger.github as any;
 const issue = gh.issue;
 const repo = gh.repository;
 
-const qWords: string[] = ["who", "what", "where", "when", "why", "how"]
-
-if (issue.title.slice(-1) == "?") {
-  addLabel()
-} else {
-  const title = issue.title.toLowerCase()
-  for (var i = 0; i < qWords.length; i++) {
-    if (title.startsWith(qWords[i])) {
-      addLabel()
-      break
-    }
+const title = issue.title.toLowerCase()
+const titleWords = title.split(" ")
+
+var labelsToAdd: string[] = []
+
+// label: question
+
+const questionWords: Set<string> = new Set(["how", "who", "what", "where", "when", "why", "which"])
+
+if (titleEndsInQuestionMark() || titleStartsWithAny(questionWords)) {
+  labelsToAdd.push("question")
+}
+
+// label: documentation
+
+const documentationWords: Set<string> = new Set(["documentation", "document", "docs", "doc", "readme", "changelog"])
```

Comment:
Do you think we should not label things as `documentation` if it has already been determined as a `question`?